src/components/Kv/KvMapLibre.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+	<div class="tw-w-full tw-relative">
+		<div :id="`kv-map-holder-${this.mapId}`" :ref="refString" style="min-height: 320px; min-width: 320px;">
+			<h1>a map</h1>
+		</div>
+	</div>
+</template>
+
+<script>
+import { createIntersectionObserver } from '@/util/observerUtils';
+import testDelayedGlobalLibrary from '@/util/timeoutUtils';
+
+export default {
+	metaInfo: {
+		script: [
+			// maplibregl - uses vector tiles and webgl for rendering
+			{ src: 'https://cdn.maptiler.com/maplibre-gl-js/v1.14.0/maplibre-gl.js', async: true, defer: true },
+		],
+		link: [
+			// maplibregl - uses vector tiles and webgl for rendering
+			{ rel: 'stylesheet', href: 'https://cdn.maptiler.com/maplibre-gl-js/v1.14.0/maplibre-gl.css' },
+		],
+	},
+	props: {
+		lat: {
+			type: Number,
+			default: 37.700091
+		},
+		long: {
+			type: Number,
+			default: -123.013243
+		},
+		/**
+		 * Setting this initialZoom will zoom the map from initialZoom to zoom when the map enters the viewport
+		 */
+		initialZoom: {
+			type: Number,
+			default: null
+		},
+		/**
+		 * Default zoom level
+		 */
+		zoomLevel: {
+			type: Number,
+			default: 4
+		},
+		/**
+		 * Set this if there are more than one map on the page
+		 */
+		mapId: {
+			type: Number,
+			default: 0
+		},
+		/**
+		 * Control how quickly the autoZoom occurs
+		 */
+		autoZoomDelay: {
+			type: Number,
+			default: 1500
+		}
+	},
+	data() {
+		return {
+			mapInstance: null,
+			mapLoaded: false,
+			zoomActive: false,
+		};
+	},
+	computed: {
+		refString() {
+			return `mapholder${this.mapId}`;
+		},
+	},
+	mounted() {
+		testDelayedGlobalLibrary('maplibregl').then(response => {
```

Comment:
I think vue-meta can handle all of this using a callback function https://vue-meta.nuxtjs.org/api/#callback

Reply:
This appears to work nicely! Thanks!

Reply:
This only works with the server injected library so wasn't feasible in a component that provides either library script.